qtav/qtav.2016-03-02.manifest

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "http://inqlude.org/schema/release-manifest-v1#",
+  "name": "qtav",
+  "release_date": "2016-03-02",
+  "version": "1.10.0",
+  "summary": "A cross-platform multimedia playback framework based on Qt and FFmpeg.",
+  "urls": {
+    "homepage": "http://www.qtav.org",
+    "vcs": "https://github.com/wang-bin/QtAV/tree/v1.10.0",
```

Comment:
I think it's better to link to the general git repo here (https://github.com/wang-bin/QtAV) rather than to the tag, because at the moment this attribute is interpreted and shown on the web page as the general URL to the git repo. I'll clarify the documentation.

We could think about introducing an additional attribute for linking to the tag, maybe something like `vcs_release_tag`. If you think that would be useful please open an issue on the inqlude repo so we can track it there.